controllers/machinedeployment_sync.go

```diff
@@ -340,7 +337,7 @@ func (r *MachineDeploymentReconciler) scale(ctx context.Context, deployment *clu
 			}
 
 			// TODO: Use transactions when we have them.
```

Comment:
Not sure what's this TODO about, could we remove it?

Reply:
Not sure either, change is from 2018 so I think we can drop it and revisit later if necessary.